### ISSUE TITLE
Is there a way to pull all data in pagination is included

### ISSUE THREAD

**Issue by seemonster1985** [EdisonJpp/react-export-html-table-to-excel#12]
Is there a way to pull all data in a set if pagination is enabled? For example, the data set is 1,000 records, we are only showing 100 records. When export is completed it only exports the 100 records showing in the table.

**Comment by EdisonJpp** [comment]
no, isn't enabled, you can use  `import { downloadExcel } from "react-export-table-to-excel";`  to export data to excel without one HTML table necessary
